CQL Translation Fails Due to Unknown Properties

When trying to translate a structured query with attribute filters into its CQL format an exception is thrown. The sq2cql library in its currently used version cannot handle additional properties unknown to it.

Related log entry:
```
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "attributeFilters" (class de.numcodex.sq2cql.model.structured_query.ConceptCriterion), not marked as ignorable (one known property: "concept"])
2022-01-26T14:18:00.606218557Z at [Source: (String)"{"version":"http://to_be_decided.com/draft-1/schema#","inclusionCriteria":[[{"termCodes":[{"code":"gender","system":"num.abide","display":"Geschlecht"}],"attributeFilters":[{"type":"concept","selectedConcepts":[{"code":"female","system":"http://hl7.org/fhir/administrative-gender","display":"Female"}],"attributeCode":{"code":"gender","system":"num.abide","display":"Geschlecht"}}],"timeRestriction":{}}]]}"; line: 1, column: 404] (through reference chain: de.numcodex.sq2cql.model.structured_query.StructuredQuery["inclusionCriteria"]->java.util.ArrayList[0]->java.util.ArrayList[0]->de.numcodex.sq2cql.model.structured_query.ConceptCriterion["attributeFilters"])
```

--- a/pom.xml
+++ b/pom.xml
@@ -130,7 +130,7 @@
     <dependency>
       <groupId>de.numcodex</groupId>
       <artifactId>sq2cql</artifactId>
-      <version>0.1.8</version>
+      <version>0.1.10</version>
     </dependency>
 
     <dependency>

